registry.json

```diff
@@ -4207,5 +4207,19 @@
    "telegram": "xbglowx",
    "operatorName": "Brian",
    "github": "xbglowx"
+  },
+  {
+   "ticker": "NOMAD",
+   "poolId": "bbebbbf81c42de5b84fbbc82c4feab78f8bd8bcf8b5af7c73a06664a",
+   "social": {
+     "website": "https://www.nomadpool.io/",
+     "twitter": "zhekree",
+     "facebook": "",
+     "instagram": "",
+     "youtube": "zhekson"
+   },
+   "telegram": "",
```

Comment:
As per the alliance requirements you must provide a valid telegram id. This must be for an individual and not a group or channel